### PR TITLE
Prepare CI for Train

### PR DESCRIPTION
**Pull request by cmurphy**
Add the Train projects to the cleanvm, repocheck, staging and submit
jobs. The Cloud:OpenStack:Train and Cloud:OpenStack:Train:Staging jobs
will need to be created and populated before these will work.

### CODE REVIEW

File: jenkins/ci.opensuse.org/openstack-cleanvm.yaml [hunk 1]
```diff
@@ -48,6 +48,16 @@
     jobs:
       - 'openstack-cleanvm-{release}'
 
+- project:
+    name: openstack-cleanvm-Train
+    disabled: false
+    release: Train
+    image:
+      - SLE_15
```
Comment (toabctl) [inline jenkins/ci.opensuse.org/openstack-cleanvm.yaml]: SLE_15_SP1

File: jenkins/ci.opensuse.org/openstack-submit.yaml [hunk 2]
```diff
@@ -42,7 +42,7 @@
             Cloud:OpenStack:[R]*)
               OSCBUILDDIST=SLE_12_SP4
             ;;
-            Cloud:OpenStack:[S]*)
+            Cloud:OpenStack:[ST]*)
```
Comment (toabctl) [inline jenkins/ci.opensuse.org/openstack-submit.yaml]: should set SLE_15_SP1

**Comment by dirkmueller** [comment]
lets first get #3725 merged and passing before starting downstream CI efforts..

**Comment by cmurphy** [comment]
Can we revisit this? The rpm-packaging CI is set up and the projects now exist in OBS.

File: jenkins/ci.opensuse.org/openstack-cleanvm.yaml (hunk 1 above)
Comment (dirkmueller) [reply jenkins/ci.opensuse.org/openstack-cleanvm.yaml]: has been addressed

**Pull request merged**